Userland/Libraries/LibWeb/DOM/Document.h

```diff
@@ -245,7 +245,7 @@ class Document
     Window& window() { return *m_window; }
 
     ExceptionOr<void> write(Vector<String> const& strings);
-    ExceptionOr<void> writeln(Vector<String> const& strings);
+    ExceptionOr<void> writeln(Vector<String>& strings);
```

Comment:
I don't understand this change, why does it need to take the Vector as a mutable reference?

Reply:
Just so we can append to it without creating a copy of the vector.
We could also `move` the list in.

Reply:
Actually we can also extract the `run the document write steps` to a static helper and just do the string building from `write` and `writeln`.